pkg/odo/cli/component/update.go

```diff
@@ -74,9 +82,15 @@ var updateCmd = &cobra.Command{
 		}
 
 		if len(componentGit) != 0 {
+<<<<<<< HEAD
 			err := component.Update(client, componentName, applicationName, "git", componentGit, stdout)
 			odoutil.CheckError(err, "")
 			log.Successf("The component %s was updated successfully", componentName)
+=======
+			err := component.Update(client, componentName, applicationName, "git", componentGit, componentGitRef, stdout)
+			util.CheckError(err, "")
+			fmt.Printf("The component %s was updated successfully\n", componentName)
+>>>>>>> added ref in create and update build configs
```

Comment:
please try to build locally using 
`make` and unit tests using`make test` before pushing 
which helps find a lot of things like easily

Reply:
my bad, forgot to save the file